test/metric_value_benchmark/eks_resources/test_schemas/cluster.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "structured log schema",
+  "description": "json schema for the cloudwatch agent k8s structured log",
+  "type": "object",
+
+  "properties": {
+    "ClusterName": {},
+    "Type": {},
+    "Sources": {},
+    "Timestamp": {},
+    "Version": {},
+    "cluster_failed_node_count": {},
```

Comment:
Why are metrics like `cluster_failed_node_count` not `required`?
We've added `namespace_number_of_running_pods` to `required` but in most of the others, we seem to leave out the metrics.